Dockerfile

```diff
@@ -1,37 +1,33 @@
 FROM kalilinux/kali-rolling:latest
 LABEL maintainer="Artis3n"
 
-RUN apt-get update && apt-get full-upgrade -y --no-install-recommends \
-    && apt-get install -y --no-install-recommends systemd seclists python3 python3-pip \
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
```

Comment:
Why the removal of `full-upgrade`?

Reply:
`full-upgrade` is the equivalent of `dist-upgrade` which really isn't something that should be done in a docker build step imo. Those should happen with updated base images, in this case in kali upstream. Even though nothing "breaks" by having it, it's effectively changing the underlying distro, which could cause side effects

Reply:
I think I'd want it for this temp environment

Reply:
I'll think about it

Reply:
realistically i doubt it would do much, since you're running off of kali latest upstream, it should just get picked up there.

honestly it was mostly just a smell for me, no strong feelings on this since it's for a dev environment.

Reply:
It does roll nightly and there's no reason I can't have an action workflow that crons a new build every night

Reply:
looking at the the previous builds that had `full-upgrade` in there, that part of it seemed to be a no-op.